packages/h5p-server/src/H5PEditor.ts

```diff
@@ -155,8 +157,10 @@ export default class H5PEditor {
             );
         }
 
-        this.globalCustomStyles =
-            this.options?.customization?.global?.styles || [];
+        this.globalCustomStyles = this.options?.customization?.global
+            ?.styles || [
```

Comment:
The theme should not replace the global styles; they should be concatenated I guess

Reply:
It is not replaced but entered as the first entry in the array .

Reply:
Really? From what I see it takes the global styles and if it is undefined it now takes the theme. They are not combined. The && doesn‘t work here, as this only works in React.